src/pages/NewChatPage.tsx

```diff
@@ -69,7 +69,8 @@ function useOptions({isGroupChat}: NewChatPageProps) {
             [],
             true,
         );
-        const maxParticipantsReached = selectedOptions.length === CONST.REPORT.MAXIMUM_PARTICIPANTS;
+        // We no longer limit the number of participants in a group chat
+        const maxParticipantsReached = false;
```

Comment:
> We can remove the limit when creating a chat. There is no limit anymore.

https://github.com/Expensify/App/issues/40512#issuecomment-2065502563

As I understand it, we have a green light to remove the relevant code altogether. Do you understand it differently? Do you think there's a chance this feature might be re-introduced?

Reply:
> We should keep the variable because we use it when we access Options:
> 
> https://github.com/Expensify/App/blob/f33217478ae8ba5ed2a2852517082a9ff9bda235/src/pages/NewChatPage.tsx#L85

In what circumstances will the `options.maxParticipantsReached` expression be truthy **after** our changes?

Reply:
Well, kind of, but then we should fix all those places... It's a standard procedure of feature removal. You start at the root (where it's obvious that some code should be removed), then follow the thread (a type checker is your friend!), adjusting all other places in the code as needed.

Ultimately, the diff can get quite big, but that's life. Unless a feature is temporarily disabled, you don't want to maintain a code full of leftovers.